lib/src/components/Banner.tsx

```diff
@@ -25,18 +83,16 @@ function	Banner({
 	secondaryButton,
 	image,
 	variant = 'default',
-	maxHeight = 'max-h-[300px]',
```

Comment:
@MarcoWorms This was bad (my fault). I removed that to handle that with a specific react ref